.github/workflows/codeql_buildscript.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+sudo apt install build-essential cmake g++-multilib libgcc-9-dev lib32gcc-9-dev ccache
```

Comment:
Should add `ninjia-build`, there was error when running `codeql_buildscript.sh`:
![image](https://github.com/bytecodealliance/wasm-micro-runtime/assets/20393758/2dc01128-293c-4320-be4f-cb3dfd76b307)
And had better add `-y` for apt install:
`sudo apt install -y build-essential cmake g++-multilib libgcc-9-dev lib32gcc-9-dev ccache ninja-build`